src/elements/tag/tag-group/tag-group.stories.ts

```diff
@@ -135,11 +150,43 @@ const AllChoiceTagGroupTemplate = ({ numberOfTagsInGroup, ...args }: Args): Temp
   </div>
 `;
 
+const TemplateWithForm = ({ numberOfTagsInGroup, ...args }: Args): TemplateResult => html`
+  <form
+    @submit=${(e: SubmitEvent) => {
+      e.preventDefault();
+      const form = (e.target as HTMLFormElement)!;
+      form.querySelector('#form-data')!.innerHTML = JSON.stringify(
+        Object.fromEntries(new FormData(form)),
+      );
+    }}
+  >
+    <sbb-tag-group ${sbbSpread(args)} style="margin-block-end: 2rem;">
+      ${repeat(new Array(numberOfTagsInGroup), (_e, i) =>
+        tagTemplate(`Label ${i + 1}`, i <= 2, `tag${i + 1}`),
+      )}
+    </sbb-tag-group>
+
+    <sbb-tag-group disabled> ${tagTemplate('Disabled tag', false)}</sbb-tag-group>
+
+    <div style="margin-block: var(--sbb-spacing-responsive-s)">
```

Comment:
from design perspective I think it's better to use an action group which introduces visual separation (inline, not margin, the margin can be kept)